Task9/backend/poll_backend/poll_backend/settings.py

```diff
@@ -0,0 +1,120 @@
+"""
+Django settings for poll_backend project.
+
+Generated by 'django-admin startproject' using Django 1.10.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.10/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.10/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.10/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'f-grg3)yx3olz=d7x75d@8-w^7gkn_2l$15m7@og=v6mt=_lq1'
```

Comment:
> SECURITY WARNING: keep the secret key used in production secret!

@AndriiKobzar I guess you should regenerate your :warning: SECRET_KEY :warning: 
There are many tools to do it:
- https://github.com/ariestiyansyah/django-secret-key
- http://www.miniwebtool.com/django-secret-key-generator/
- https://gist.github.com/ndarville/3452907